lib/generators/boring/sentry/install/install_generator.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module Boring
+  module Sentry
+    class InstallGenerator < Rails::Generators::Base
+      desc 'Adds sentry to the app'
+
+      def add_sentry_gems
+        say 'Adding sentry gem to Gemfile', :green
+
+        gem 'sentry-ruby'
+        gem 'sentry-rails'
+      end
+
+      def configure_sentry_gem
+        say 'Configuring sentry gem', :green
+
+        sentry_config = <<~RUBY
+          Sentry.init do |config|
+            config.dsn = Rails.application.credentials.dig(:sentry, :dsn_key)
+            # enable performance monitoring
+            config.enable_tracing = true
+            # get breadcrumbs from logs
+            config.breadcrumbs_logger = [:active_support_logger, :http_logger]
+          end
+        RUBY
+
+        say 'Please add your dsn key to your rails credentials under sentry.dsn_key', :yellow
```

Comment:
Let's notify about these things separately at the end of the generator (by creating new method) so they are not missed by the user.

Reply:
Also message could be a bit better, you can take a look at this example from another generator:

```
say "❗️❗️\nusername will be used from `ENV.fetch('SWAGGER_UI_LOGIN_USERNAME', 'admin')` and password from `Rails.application.credentials.dig(:swagger_ui, :password)`. You can change these values if they don't match with your app.\n", :yellow
```